assets/styles/style.css

```diff
@@ -0,0 +1,188 @@
+:root {
+  /* color */
+  --clr-primary: hsl(222, 11%, 17%);
+  --clr-secondary: hsl(7, 84%, 58%);
+  --clr-primary-light: hsl(0, 0%, 27%);
+  --clr-neutral: hsl(0, 0%, 83%);
+  --clr-neutral-light: hsl(0, 0%, 95%);
+  --clr-white: hsl(0, 0%, 100%);
+
+  /* fonts */
+  --ff-family: 'Lato', sans-serif;
+  --ff-heading: var(--ff-family);
+  --ff-body: var(--ff-family);
+  --fw-regular: 400;
+  --fw-regurlar-plus: 500;
+  --fw-semi-bold: 700;
+  --fw-bold: 900;
```

Comment:
- [ ] Please 🙏🏽 use the Cocogoose font family to be similar to the original design.

Reply:
Good point @hansderly don't worry I will turn it to an optional requirement.

Reply:
Thanks @franclobo